src/gwt/src/org/rstudio/studio/client/workbench/views/vcs/common/diff/LineTableView.java

```diff
@@ -148,8 +149,8 @@ public void render(Context context, ChunkOrLine value, SafeHtmlBuilder sb)
                      sb,
                      RES.cellTableStyle().lineActions(),
                      selectionModel_.getSelectedSet().size() > 1
-                     ? " selection"
-                     : " line");
+                     ? constants_.selectionSuffix("")
```

Comment:
Is this technique (passing an empty string to placeholder) required to have a leading space at the start of the localizable string?

`selectionSuffix={0} selection`

Guessing you can't do this?

`selectionSuffix= selection`

Reply:
Yup that's correct, doing `selectionSuffix= selection` will just have `selectionSuffix` resolving to `selection` with no leading space. If you would prefer, we could change it to be `" " + constants_.selectionSuffix()`.

Reply:
It's fine as-is. I've not worked with Java property files before so wasn't sure.